src/node/test/common_test.js

```diff
@@ -98,27 +99,23 @@ describe('Proto message long int serialize and deserialize', function() {
   });
 });
 describe('Proto message bytes serialize and deserialize', function() {
-  var sequenceSerialize = common.serializeCls(messages_proto.SequenceValues);
+  var sequenceSerialize = common.serializeCls(messages_proto.lookup('SequenceValues'));
   var sequenceDeserialize = common.deserializeCls(
-      messages_proto.SequenceValues);
-  var sequenceBase64Deserialize = common.deserializeCls(
-      messages_proto.SequenceValues, true);
+      messages_proto.lookup('SequenceValues'));
   var buffer_val = new Buffer([0x69, 0xb7]);
   var base64_val = 'abc=';
   it('should preserve a buffer', function() {
-    var serialized = sequenceSerialize({bytes_field: buffer_val});
+    var serialized = sequenceSerialize({bytesField: buffer_val});
```

Comment:
In that case, can you add an additional test to use the camel case names, instead of modifying the existing tests?